package.json

```diff
@@ -31,9 +31,9 @@
   },
   "optionalDependencies": {
     "get-pixels": "*",
-    "serialport": "*",
     "qr-image": "*",
-    "usb": "*"
+    "serialport": "*",
+    "usb": "git+https://github.com/yhauxell/node-usb.git"
```

Comment:
Why would you want to use the latest usb module without using a stable version of npm?